scripts/examples/generate_book_cover.py

```
@@ -0,0 +1,144 @@
+                if message['data']['prompt_id'] == prompt_id:
+                    node_output = message['data']['output']
+                    if 'images' in node_output:
+                        node_id = message['data']['node']
+                        output_images[node_id] = node_output['images']
+        else:
+            continue
+
+    print("Generation complete.")
+
+    # 5. Download and save images locally
+    for node_id, images in output_images.items():
```

Ruff (0.15.1)

[warning] Loop control variable `node_id` not used within loop body

(B007)